miner/scroll_worker.go

```diff
@@ -561,8 +562,13 @@ func (w *worker) handleForks() (bool, error) {
 // processTxPool
 func (w *worker) processTxPool() (bool, error) {
 	tidyPendingStart := time.Now()
-	// Fill the block with all available pending transactions.
-	pending := w.eth.TxPool().PendingWithMax(false, w.config.MaxAccountsNum)
+
+	pending := w.pendingTxns
```

Comment:
Hmm, ownership of pending gets transferred to `TransactionsByPriceAndNonce` once `NewTransactionsByPriceAndNonce` is called. So we can't just reuse the map as a whole without cloning it first.